src/video/sdlgenblit.pl

```diff
@@ -438,7 +438,7 @@ sub output_copyfunc
 
     $sa =~ s/[A8]//g;
     $da =~ s/[A8]//g;
-    $matching_colors = (!$modulate && !blend && ($sa eq $da)) ? 1 : 0;
+    $matching_colors = (!$modulate && !$blend && ($sa eq $da)) ? 1 : 0;
```

Comment:
Looked at it for minutes, and hadn't noticed the missing `$`, sigh..

Reply:
The curse of working with rich IDEs, right?